.ci/validation.sh

```diff
@@ -1251,6 +1251,32 @@ sevntu)
   ./mvnw -e --no-transfer-progress clean compile checkstyle:check@sevntu-checkstyle-check
   ;;
 
+openrewrite-recipes)
+  echo "Cloning and building OpenRewrite recipes..."
+  PROJECT_ROOT="$(pwd)"
+  export MAVEN_OPTS="-Xmx4g -Xms2g"
+
+  cd /tmp
+  git clone https://github.com/checkstyle/checkstyle-openrewrite-recipes.git
+  cd checkstyle-openrewrite-recipes
+  mvn -e --no-transfer-progress clean install -DskipTests
+
+  cd "$PROJECT_ROOT"
+
+  echo "Running Checkstyle validation to get report for openrewrite..."
+  set +e
+  ./mvnw -e --no-transfer-progress clean compile antrun:run@ant-phase-verify
+  set -e
+  echo "Running OpenRewrite recipes..."
+  ./mvnw -e --no-transfer-progress -Drewrite.recipeChangeLogLevel=INFO \
+  rewrite:run -P checkstyle-autofix
+
+  echo "Checking for uncommitted changes..."
```

Comment:
in rare cases it would fail, laking flaky state to prod.

would need to run twice to really make sure, as big changes/recipes sometimes need multiple runs mostly twice.

17 migration changes from `javax` to `jarkata` having the need to reorder/run, after initial run/migration. 

- https://github.com/checkstyle/checkstyle/pull/17730